source/extensions/common/wasm/context.h

```diff
@@ -374,6 +374,19 @@ class Context : public proxy_wasm::ContextBase,
   Http::HeaderMap* getMap(WasmHeaderMapType type);
   const Http::HeaderMap* getConstMap(WasmHeaderMapType type);
 
+  void onHeadersModified(WasmHeaderMapType type) {
+    if (type == WasmHeaderMapType::RequestHeaders) {
+      if (!no_automatic_route_refresh_) {
```

Comment:
nit: no double negation please for readability

Reply:
got it.